app/javascript/shared/components/Checkbox.re

```diff
@@ -3,9 +3,15 @@
 let str = React.string;
 
 [@react.component]
-let make = (~id, ~label, ~onChange) => {
+let make = (~id, ~label, ~onChange, ~checked=false) => {
   <div>
-    <input className="hidden checkbox-input" id type_="checkbox" onChange />
+    <input
+      className="hidden checkbox-input"
```

Comment:
Where is `checkbox-input` styled? Where-ever the styling is, it should probably be pulled in to a separate CSS here? 🤔

Reply:
Its a seperate file `shared/components/Checkbox.css`